### PR TITLE
WIP: ONNX Exportable

### PR DESCRIPTION
**Pull request by briacht**
Add ONNX exportable trainers and transforms in existing docs

Preview links:

File Type | File Name | Published Url
-- | -- | --
Content | docs/machine-learning/how-to-choose-an-ml-net-algorithm.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-choose-an-ml-net-algorithm?branch=pr-en-us-23573
Content | docs/machine-learning/resources/transforms.md | https://review.docs.microsoft.com/en-us/dotnet/machine-learning/resources/transforms?branch=pr-en-us-23573

### CODE REVIEW

**Comment by briacht** [comment]
I added some algorithms which were missing, but I'll need someone with more data science expertise to help with the "why choose this algo" part.

File: docs/machine-learning/resources/transforms.md [hunk 1]
```diff
@@ -20,18 +20,18 @@ Other data transformations don't require training data. For example: the <xref:M
...
+| <xref:Microsoft.ML.TransformExtensionsCatalog.CopyColumns%2A> | Copy and rename one or more input columns | Yes |
+| <xref:Microsoft.ML.TransformExtensionsCatalog.DropColumns%2A> | Drop one or more input columns | No |
+| <xref:Microsoft.ML.TransformExtensionsCatalog.SelectColumns%2A> | Select one or more columns to keep from the input data | Yes |
 
 ## Normalization and scaling
 
-| Transform | Definition |
-| --- | --- |
-| <xref:Microsoft.ML.NormalizationCatalog.NormalizeMeanVariance%2A> | Subtract the mean (of the training data) and divide by the variance (of the training data) |
+| Transform | Definition | ONNX Exportable |
```
Comment (luisquintanilla) [inline docs/machine-learning/resources/transforms.md:32]: Are normalization / scaling exportable to ONNX?

**Review by luisquintanilla — COMMENTED** [review]
Looks great! Once you get feedback on why choose certain algorithms it should be good to merge.

File: docs/machine-learning/resources/transforms.md (hunk 1 above)
Comment (michaelgsharp) [reply docs/machine-learning/resources/transforms.md:32]: They are. Bri asked and I double checked in the code itself. It looks like not all the xml docs in the machine learning repo itself are up to date.

**Comment by briacht** [comment]
@luisquintanilla  Maybe we should merge for now, and we can create a new PR for updating trainers and "how to choose an algo"?

File: docs/machine-learning/how-to-choose-an-ml-net-algorithm.md [hunk 2]
```diff
@@ -54,50 +91,144 @@ Decision tree algorithms take more resources and do not scale as well as linear
...
+
+### Field Aware Factorization Machine
+
+ Best for sparse categorical data, with large datasets.
+
+|Trainer|Task|ONNX Exportable|
+|---------|----------|----------|
+|<xref:Microsoft.ML.Trainers.FieldAwareFactorizationMachineTrainer>|Binary classification|No|
 
 ## Meta algorithms
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-choose-an-ml-net-algorithm.md:152]: @briacht are the meta algos also going into a table?

Comment (briacht) [reply docs/machine-learning/how-to-choose-an-ml-net-algorithm.md:152]: Which ones? Ova and Pairwise are already in tables

**Review by luisquintanilla — APPROVED** [review]

**Pull request merged**